tests/integration/test_sqs.py

```diff
@@ -861,6 +865,39 @@ def test_create_queue_with_slashes(self):
         result = self.client.list_queues()
         self.assertNotIn(queue_url.get('QueueUrl'), result.get('QueueUrls'))
 
+    def test_list_queue_with_auth_in_presigned_url(self):
+        base_url = '{}://{}:{}'.format(get_service_protocol(), config.LOCALSTACK_HOSTNAME, config.PORT_SQS)
+        # encoded_url = urlencode()
+        req = AWSRequest(method='GET', url=base_url, params={
+            'Action': 'ListQueues',
+            'Version': '2012-11-05'
+        })
+
+        # boto doesn't support querystring-style auth, so we have to do some
+        # weird logic to use boto's signing functions, to understand what's
+        # going on here look at the internals of the SigV4Auth.add_auth
+        # method.
+        datetime_now = datetime.datetime.utcnow()
+        req.context['timestamp'] = datetime_now.strftime(SIGV4_TIMESTAMP)
+        signer = SigV4Auth(Credentials('test', 'test'), 'sqs', 'us-east-1')
```

Comment:
Let's better use the `TEST_AWS_ACCESS_KEY_ID`/`TEST_AWS_SECRET_ACCESS_KEY` constants from `constants.py`, and `get_region()` rather than hardcoding the region here:
```
     signer = SigV4Auth(Credentials(TEST_AWS_ACCESS_KEY_ID, TEST_AWS_SECRET_ACCESS_KEY), 
         'sqs', aws_stack.get_region())
```